src/org/daisy/dotify/formatter/impl/VolumeProvider.java

```diff
@@ -224,14 +225,16 @@ public double getCost(SplitPointDataSource<Sheet, ?> units, int index, int break
 
 		pageIndex += pageCount;
 		SectionBuilder sb = new SectionBuilder();
+        boolean atFirstPageOfContents = true;
 		for (Sheet sheet : contents) {
 			for (PageImpl p : sheet.getPages()) {
 				for (String id : p.getIdentifiers()) {
-					crh.setVolumeNumber(id, volumeNumber);
+                    crh.setVolumeData(id, new VolumeData(volumeNumber, atFirstPageOfContents));
```

Comment:
I changed everything to TABs.

Reply:
I don't think it's a good idea to mix the white space changes with the code changes though. Shouldn't we make one clean up commit for the whole code base, and do the rest before or after? Otherwise it will be hard to extract what was actually done in this PR later when we want to view the history. And also with one global clean up commit we don't have the same issues with future PRs.

Reply:
When we are going to do one big clean-up commit, then my preference would be to remove all those TABs and replace them with 4 spaces each. That is how we (me and my colleagues) work in all other projects.
What happened to me is that I started to work in Dotify and I forgot to change the setting in the editor to use TABs instead of spaces.

Reply:
OK I also have my preference, but I'm flexible. Let's discuss this further offline.